src/components/ReportActionItem/ReportPreview.js

```diff
@@ -99,7 +99,7 @@ const ReportPreview = (props) => {
     const managerName = ReportUtils.isPolicyExpenseChat(props.chatReport) ? ReportUtils.getPolicyName(props.chatReport) : ReportUtils.getDisplayNameForParticipant(managerEmail, true);
     const isCurrentUserManager = managerEmail === lodashGet(props.session, 'email', null);
     return (
-        <View style={[styles.chatItemMessage, styles.mt4]}>
```

Comment:
Hi ✋ Coming from https://github.com/Expensify/App/issues/22797

We must maintain the top margin when previewing with the user details avatar. Instead of removing this style, we should pass a new props style `8px` margin from `ReportActionItem` based on the message not grouped with the previous message.